src/CMakeLists.txt

```diff
@@ -434,32 +435,31 @@ if(WITH_JAEGER)
 		    ${CMAKE_BINARY_DIR}/external/lib/libyaml-cpp.so.0.6.2)
     execute_process(COMMAND bash -c "grep -q 'yaml-cpp' debian/libjaeger.install || echo 'usr/lib/libyaml-cpp.so.*' >> debian/libjaeger.install"
 		    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
-    add_library(yaml-cpp::yaml-cpp SHARED IMPORTED)
-    add_dependencies(yaml-cpp::yaml-cpp yaml-cpp)
     set_library_properties_for_external_project(yaml-cpp::yaml-cpp yaml-cpp)
     list(APPEND dependencies "yaml-cpp")
+    set(jaeger_base yaml-cpp::yaml-cpp)
   endif()
-  include(IncludeJaeger)
-  add_library(jaeger-base INTERFACE)
-  add_dependencies(common-objs
-  yaml-cpp::yaml-cpp
-  opentracing::libopentracing
-  thrift::libthrift
-  jaegertracing::libjaegertracing)
-  target_link_libraries(jaeger-base INTERFACE
-  yaml-cpp::yaml-cpp
+
+  include(BuildJaeger)
+  set(jaeger_base ${jaeger_base}
   opentracing::libopentracing
-  thrift::libthrift
   jaegertracing::libjaegertracing)
-  include_directories(SYSTEM ${CMAKE_BINARY_DIR}/external/include)
-  #with CMake 3.12+ the following can be replaced by:
+  message(" jaeger build targets: " ${jaeger_base})
```

Comment:
per CMake document

> (none) or NOTICE
>    Important message printed to stderr to attract user's attention.

why do you think we should print this using `NOTICE`?

Reply:
since the externalProjects are being build conditionally, I thought it would be easier to know which ones of them would be used. But I can drop this, once PR tests passes.